public/index.html

```diff
@@ -35,6 +35,20 @@
     <script src="/modernizr.min.js"></script>
     <script src="https://polyfill.io/v3/polyfill.min.js?features=Array.from,Array.isArray,Array.prototype.entries,Array.prototype.every,Array.prototype.fill,Array.prototype.filter,Array.prototype.find,Array.prototype.findIndex,Array.prototype.flat,Array.prototype.forEach,Array.prototype.includes,Array.prototype.indexOf,Array.prototype.keys,Array.prototype.lastIndexOf,Array.prototype.map,Array.prototype.reduce,Array.prototype.some,Array.prototype.values,ArrayBuffer,Blob,console,DataView,Date.now,Date.prototype.toISOString,document,Element,Float32Array,Function.prototype.bind,getComputedStyle,Intl,JSON,Map,Math.log10,Math.sign,modernizr:es5object,modernizr:es6string,MutationObserver,Number.EPSILON,Number.isNaN,Number.MAX_SAFE_INTEGER,Number.MIN_SAFE_INTEGER,Object.assign,Object.entries,Object.freeze,Object.getOwnPropertyDescriptors,Object.getOwnPropertySymbols,Object.is,Object.isExtensible,Object.isFrozen,Object.setPrototypeOf,Object.values,Promise,Reflect,Reflect.construct,requestAnimationFrame,Set,String.prototype.trim,String.prototype.trimStart,Symbol,Symbol.for,Symbol.iterator,Symbol.prototype.description,Symbol.toPrimitive,Symbol.toStringTag,URLSearchParams,WeakMap,WeakSet,XMLHttpRequest"></script>
     <script src="/analytics.js"></script>
+
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
```

Comment:
It would be cool to not load these on every page, or to load them all with `defer`, but I think that without lazy loading, we can't split them out meaningfully @red-tt-bear

Reply:
We could add a hook to insert the script tag wherever Quill was used, not sure if that would get messy in any way.